asserts/system_user.go

```diff
@@ -216,6 +235,18 @@ func checkHashedPassword(headers map[string]interface{}, name string) (string, e
 	return pw, nil
 }
 
+func checkSystemUserExpiration(headers map[string]interface{}) (string, error) {
+	str, err := checkOptionalString(headers, "user-valid-for")
```

Comment:
this will not do the correct thing if user-valid-for is not there at all. For this kind of field btw we tend to have dedicated test called TestUserValidForIsOptional (see example of such tests in model_test.go)

Reply:
I updated this now! Thanks for pointing this out!